lucasgmelo/cinema-management
# Feat/navbar

## Description

I created the navbar component

<img width="872" alt="image" src="https://user-images.githubusercontent.com/61155203/185812779-d4cbd681-91ed-4bbd-b1c0-a37050457e9a.png">

Created a service to handle these states and functions.
Also added a logout button (didn't exist)
Implemented auto login
Created unit tests

## How to test

`git pull`
in terminal, type `cd client`
then type `npm install`,
`npm run start`

you should be seeing the sidebar and a text 'test'.
if you click the button, you'll be seeing that the name has changed and now there's another text 'Gerencias'
if you are logged and reload, next time your name will be there already
if you click the logout button, you're a guest again

for tests
type `npm run test`

probably, setup errors are going to appear. talk to me and i'll help you to fix it.

## Type of change

Please delete options that are not relevant.


## Checklist:
